ui/src/Constant/constant.js

```diff
@@ -1,5 +1,11 @@
 export const FOOTER = {
-    "COPYRIGHT": "Copyright 2022-2023",
-    "RDS_WEBSITE": "RDS Website",
-    "DISCORD_SERVER":"Discord server"
-}
\ No newline at end of file
+  COPYRIGHT: "Copyright 2022-2023",
```

Comment:
I didn't want to edit this, but I guess my formatter formatted this whole document.